compiler/next/test/frontend/testParse.cpp

```diff
@@ -287,6 +289,172 @@ static void test15(Parser* parser) {
   assert(block->stmt(2)->isComment());
 }
 
+static void testCalls1(Parser* parser) {
```

Comment:
Just curious, we can't build the AST for `f(g())` yet, because we are still only looking at top-level statements, right?

Reply:
I think `f(g())` might be parsable today.

Reply:
I added a test case for `f(g(x))``.